packages/client/modules/email/components/NotificationSummaryEmailRoot.tsx

```diff
@@ -0,0 +1,74 @@
+import graphql from 'babel-plugin-relay/macro'
+import ms from 'ms'
+import {NotificationSummaryEmailRootQuery} from 'parabol-client/__generated__/NotificationSummaryEmailRootQuery.graphql'
+import React from 'react'
+import {QueryRenderer} from 'react-relay'
+import {Environment} from 'relay-runtime'
+import {typePicker} from './EmailNotifications/EmailNotificationPicker'
+import NotificationSummaryEmail from './NotificationSummaryEmail'
+
+const query = graphql`
+  query NotificationSummaryEmailRootQuery($first: Int!, $after: DateTime) {
+    viewer {
+      notifications(first: $first, after: $after)
+        @connection(key: "NotificationDropdown_notifications") {
+        edges {
+          node {
+            id
+            createdAt
+            status
+            type
+            ...EmailNotificationPicker_notification
+          }
+        }
+      }
+      id
+    }
+  }
+`
+
+export interface NotificationSummaryRootProps {
+  appOrigin: string
+  preferredName: string
+  notificationCount: number
+  environment: Environment
+}
+
+const NotificationSummaryEmailRoot = (props: NotificationSummaryRootProps) => {
+  const {appOrigin, preferredName, notificationCount, environment} = props
+
+  return (
+    <QueryRenderer<NotificationSummaryEmailRootQuery>
+      environment={environment}
+      query={query}
+      variables={{first: 30, after: new Date().toISOString()}}
```

Comment:
+1 let's make `30` a const to avoid magic numbers and make it clearer how many notifications will be displayed within the email

Reply:
Done